uniffi_bindgen/src/bindings/kotlin/templates/Helpers.kt

```diff
@@ -77,6 +77,7 @@ private inline fun <U> rustCall(callback: (RustCallStatus) -> U): U {
 public class USize(value: Long = 0) : IntegerType(Native.SIZE_T_SIZE, value, true) {
     // This is needed to fill in the gaps of IntegerType's implementation of Number for Kotlin.
     override fun toByte() = toInt().toByte()
+    @Deprecated("`toInt().toChar()` is deprecated")
```

Comment:
Can you link to the jetbrains issue that will make this unneeded?

Reply:
https://github.com/mozilla/uniffi-rs/issues/1685

We should get that fixed outside of this lengthy PR I guess

Reply:
That makes a lot of sense, #1692 does this